src/link/link.tsx

```diff
@@ -76,3 +94,9 @@ const RouterLink = (props: React.AnchorHTMLAttributes<HTMLAnchorElement>) => (
         {props.children}
     </Link>
 );
+
+export const ExternalRouterLink = (props: React.AnchorHTMLAttributes<HTMLAnchorElement>) => (
```

Comment:
Tanken med `External` og `Router` i navnene var at `External`-lenker går til eksterne sider og derfor ikke har noe med `react-router` å gjøre, mens `Router`-lenker er interne lenker som må håndteres spesielt fordi de bare skal gå til en annen side innenfor samme `Router`.

Dette er en `External`-lenke uten styling?

Reply:
> Tanken med `External` og `Router` i navnene var at `External`-lenker går til eksterne sider og derfor ikke har noe med `react-router` å gjøre, mens `Router`-lenker er interne lenker som må håndteres spesielt fordi de bare skal gå til en annen side innenfor samme `Router`.
> 
> Dette er en `External`-lenke uten styling?

Ok, jeg skjønner, ja dette er en `External` uten styling. Jeg kan godt kalle den noe annet, men den var veldig nyttig for inngangssiden i alle fall.

Reply:
Ja, det skjønner jeg. Tenkte bare på navnet.
Det kanskje lurt å omstrukturere eksisterende komponenter litt.
Det som er nå passer ikke nødvendigvis med det som trengs.

Reply:
Endret navnet på min i alle fall. Kanskje ha omstrukturering i en separat PR?

Reply:
Ja, vi kan vurdere det. Synes det ser bra ut nå :)